src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -663,7 +663,9 @@ def forward(
             hidden_states[~attention_mask] = 0.0
 
             # extend attention_mask
-            attention_mask = (1.0 - attention_mask[:, None, None, :].to(dtype=hidden_states.dtype)) * -10000.0
+            attention_mask = (1.0 - attention_mask[:, None, None, :].to(dtype=hidden_states.dtype)) * torch.finfo(
```

Comment:
(nit) same - maybe nicer to have it in two lines. But feel free to disregard if it's too much work